scripts/prepare_lambda_deployment.py

```diff
@@ -34,6 +36,11 @@ def transform_tf(input_json):
         assert 'layers' not in func
         func['layers'] = ["${var.layer_arn}"]
 
+        # Inject runtime ES environment variables provisioned by Terraform
+        for var, val in config.es_endpoint_env(es_endpoint=('${var.es_endpoint[0]}', '${var.es_endpoint[1]}'),
+                                               es_instance_count='${var.es_instance_count}').items():
+            func['environment']['variables'][var] = val
```

Comment:
Assert that the variable isn't already set before setting it.

Reply:
Asserting that 'AZUL_ES_INSTANCE_COUNT' variable isn't present for deployments not sharing ES domain with `dev` might fail. That variable is required to be present in `newdeployment.local/environment.py` in order to create a ES instance for a new deployment. Should the assertion be selective on whether the deployment shares an ES domain with `dev` ?

Reply:
It is a bit awkward but I have no better idea. Scratch the assert.